updater/lib/dependabot/api_client.rb

```diff
@@ -119,13 +119,10 @@ def update_dependency_list(dependencies, dependency_files)
       sleep(rand(3.0..10.0)) && retry
     end
 
-    def record_package_manager_version(ecosystem, package_managers)
-      api_url = "#{base_url}/update_jobs/#{job_id}/record_package_manager_version"
+    def record_ecosystem_versions(ecosystem_versions)
```

Comment:
Just making sure I've thought about this properly: this endpoint is new, so if someone pulls down an image with this change to GHES the service might not have that endpoint but that's fine as we've de-fanged the call unless an experiment is enabled in the [previous PR](https://github.com/dependabot/dependabot-core/pull/7516) and it will never been enabled for any version of GHES.

Reply:
Exactly.